dev/Makefile

```diff
@@ -2,34 +2,37 @@ EXPECTED_CONTEXT=colima
 NAMESPACE=optimus-dev
 OPTIMUS_ODPF_CHART=odpf/optimus
 
+DAGS_PATH?=./dags
```

Comment:
using abspath is decent but only `$HOME/*` and `/tmp/colima/*` will work. Other paths won't work.

Reply:
I made an assumption that user by default always create a folder inside `$HOME/*`, and any relative path will be expanded to absolute path on this loc https://github.com/odpf/optimus/blob/57725f11e1a216b7f8a3aa8baaff87a6c28946b4/dev/Makefile#L20